src/python/WMCore/WMSpec/Steps/Executors/LogCollect.py

```diff
@@ -106,6 +106,11 @@ def execute(self, emulator=None):
             try:
                 if int(result.group(1)) >= 8:
                     useEdmCopyUtil = True
+                elif int(result.group(1)) < 8 and scramArch.startswith('sl6_amd64_'):
+                    useEdmCopyUtil = True
+                    logging.warning("WARNING: CMSSW too old to support edmCopyUtil, using CMSSW_10_2_3 instead")
```

Comment:
This `WARNING` in the message body can go away. Logging should take care and record the log level altogether.